requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 tox
 pytest
 treon
-pylint
+black==22.1.0
```

Comment:
Is there a reason why we are specifying such a concrete version of black as opposed to what we do with other requirements?